integration-tests/environments/react-native-test-app/package.json

```diff
@@ -0,0 +1,150 @@
+      }
+    },
+    "test:ios": {
+      "command": "mocha-remote --reporter @realm/mocha-reporter -- concurrently --kill-others-on-fail npm:metro npm:runner",
+      "dependencies": [
+        "pod-install:simulator",
+        "../../../packages/realm:build:ts",
+        "../../../packages/mocha-reporter:bundle"
+      ],
+      "env": {
+        "PLATFORM": "ios"
+      }
+    },
+    "test:ci:android": {
+      "command": "mocha-remote -- concurrently --kill-others-on-fail npm:metro npm:runner",
+      "env": {
+        "PLATFORM": "android"
+      }
+    },
+    "test:ci:ios": {
+      "command": "mocha-remote -- concurrently --kill-others-on-fail npm:metro npm:runner",
+      "dependencies": [
+        "pod-install:ci"
+      ],
+      "env": {
+        "PLATFORM": "ios"
+      }
+    },
+    "pod-install:simulator": {
+      "command": "pod-install",
+      "dependencies": [
+        {
+          "script": "../../../packages/realm:build:ios:debug:simulator",
+          "cascade": false
+        }
+      ],
+      "clean": "if-file-deleted",
+      "files": [
+        "ios/Podfile",
+        "../../../packages/realm/react-native/ios/realm-js-ios.xcframework"
+      ],
+      "output": [
+        "ios/Pods",
+        "ios/Podfile.lock"
+      ],
+      "env": {
+        "USE_HERMES": {
+          "external": true,
+          "default": "1"
+        },
+        "RCT_NEW_ARCH_ENABLED": {
+          "external": true,
+          "default": "1"
+        }
+      }
+    },
+    "pod-install:ci": {
+      "command": "pod-install",
+      "env": {
+        "USE_HERMES": {
+          "external": true,
+          "default": "1"
+        },
+        "RCT_NEW_ARCH_ENABLED": {
+          "external": true,
+          "default": "1"
+        }
+      }
+    }
+  },
+  "dependencies": {
+    "base-64": "^1.0.0",
+    "mocha-remote-react-native": "^1.12.2",
+    "react": "18.2.0",
+    "react-native": "0.73.6",
+    "react-native-fetch-api": "^3.0.0",
+    "react-native-get-random-values": "^1.11.0",
+    "realm": "*",
+    "text-encoding": "^0.7.0",
+    "web-streams-polyfill": "^4.0.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.20.0",
+    "@babel/preset-env": "^7.20.0",
+    "@babel/runtime": "^7.20.0",
+    "@react-native/babel-preset": "0.73.19",
+    "@react-native/eslint-config": "0.73.2",
+    "@react-native/metro-config": "0.73.3",
+    "@react-native/typescript-config": "0.73.1",
+    "@rnx-kit/metro-config": "^1.3.14",
+    "@types/react": "^18.2.6",
+    "@types/react-test-renderer": "^18.0.0",
+    "concurrently": "^8.2.2",
+    "eslint": "^8.19.0",
+    "mkdirp": "^1.0.0",
+    "mocha-remote-cli": "^1.12.2",
+    "path-browserify": "^1.0.1",
+    "pod-install": "^0.2.0",
+    "prettier": "2.8.8",
+    "react-native-fs": "^2.20.0",
+    "react-native-polyfill-globals": "^3.1.0",
+    "react-native-test-app": "^3.2.16",
+    "react-test-renderer": "18.2.0",
+    "typescript": "5.0.4"
+  },
+  "engines": {
+    "node": ">=18"
```

Comment:
Should it be >=20? I mean, 18 is in maintenance mode

Reply:
We could probably leave it out, as we already have `engines` in our root package.json.